mqtt/mqtt-broker/src/connection.rs

```diff
@@ -97,7 +96,7 @@ where
     match codec.next().await {
         Some(Ok(Packet::Connect(connect))) => {
             let client_id = client_id(&connect.client_id);
-            let (sender, events) = mpsc::channel(128);
+            let (sender, events) = mpsc::unbounded_channel();
```

Comment:
It can cause infinite growth of message queue, right?

Reply:
yes, but in this case we are bounding it with the session (Dequeue and number of inflight messages). Generally, I would be scared about an unbounded queue. However, in this case we have a different mechanism to bound the number of messages.

The key here is that we cannot block this loop. So, the send needs to not block.

Reply:
Yeah, that's what I figured. But worth to ask!